apps/prairielearn/src/tests/api.test.js

```diff
@@ -60,20 +60,22 @@ describe('API', function () {
       const button = page$('#generateTokenButton').get(0);
 
       // Load the popover content
-      assert.isString(button.attribs['data-content']);
+      assert.isString(button?.attribs['data-content']);
+      if (!button) return;
```

Comment:
I don't think we want to just return here; can we instead `assert(button)` so that the test fails instead of silently succeeding?

Reply:
That works (done in db8f17f). I had tried `assert.isDefined` or `assert.isNotNull`, but that didn't satisfy the typechecking.